taxi/forms.py

```diff
@@ -0,0 +1,37 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+
+    class Meta:
+        model = Driver
+        fields = ("license_number",)
+
+    def clean_license_number(self):
+        return validate_license_number(self.cleaned_data["license_number"])
+
+
+def validate_license_number(license_number,):
+    if len(license_number) != 8:
+        raise ValidationError("License number should consist of 8 characters")
+    elif not license_number[:3].isupper() or not license_number[:3].isalpha():
+        raise ValidationError("First 3 characters should be uppercase letters")
+    elif not license_number[3:].isdigit():
+        raise ValidationError("Last 5 characters should be digits")
+
+    return license_number
```

Comment:
You can move this directly to `clean_licence_number`, no need to create additional function